plume-file-core/src/main/java/com/coreoz/plume/file/services/FileService.java

```diff
@@ -143,12 +143,29 @@ public Optional<FileMetadata> fetchMetadata(String fileUniqueName) {
      */
     public void deleteUnreferenced() throws UncheckedIOException {
         List<String> fileUniqueNamesToDelete = fileMetadataService.findUnreferencedFiles();
+        this.deleteFilesByUniqueName(fileUniqueNamesToDelete);
+        logger.debug("{} unreferenced files deleted", fileUniqueNamesToDelete.size());
+    }
+
+    /**
+     * Delete files from types that longer exists in the {@link FileType} enum and can't be linked to stored files
+     * This method must be use with caution and punctually, as it can lead to errors
```

Comment:
Delete files having a type that longer exists in the {@link FileType} enum and can't be linked to stored files.
This method should be used punctually only when a file type is deleted. Else unwanted files could be deleted in a shared metadata database environment.